backend/pkg/modules/internal/traceanalyzer/sensitive/findings.go

```diff
@@ -103,15 +103,6 @@ func (a APIAnnotationRegexpMatching) Redacted() utils.TraceAnalyzerAPIAnnotation
 	return &newA
 }
 
-func (a *APIAnnotationRegexpMatching) ToFinding() utils.Finding {
-	return utils.Finding{
-		ShortDesc:    "Matching regular expression",
```

Comment:
Is this your internal format? So Short Desc==Name?

Reply:
It was the previous format, served for the UI.
Now, we are using APIFindings in the UI.